### PR TITLE
Issue #451: Swipe to go back is not working

### PR DESCRIPTION
**Pull request by iosjillian**
* Fixes Issue #451 
* Removed custom back navigation
* Set up navigation destinations
* Updated README
* Removed cocoapods from Gemfile

Here's what it looks like! Not sure why there's a circle on the Chevron, maybe that's a new default? I'm happy to make updates to the UI if desired. This also changed the opacity/scroll behavior of the nav bar a bit to the default.

https://github.com/user-attachments/assets/3555fff1-49d5-46a7-a06c-b5658c7b4fc3

### CODE REVIEW

**Review by cselin — APPROVED** [review]
Nice work!

File: FiveCalls/FiveCalls/IssueDone.swift [hunk 1]
```diff
@@ -115,7 +115,6 @@ struct IssueDone: View {
             }
             .padding(.horizontal)
         }
-        .navigationBarHidden(true)
```
Comment (nickoneill) [inline FiveCalls/FiveCalls/IssueDone.swift:118]: Let's keep this line, I don't think you should be able to go back from the done page

**Review by nickoneill — APPROVED** [review]
Good stuff! The custom header was not doing enough to be worth the extra complexity

File: FiveCalls/FiveCalls/IssueDone.swift (hunk 1 above)
Comment (iosjillian) [reply FiveCalls/FiveCalls/IssueDone.swift]: Good catch! Updated

**Review by nickoneill — APPROVED** [review]
excellent, thank you!

**Pull request merged**